build.gradle

```diff
@@ -345,6 +345,7 @@ tasks.withType(Test) {
                 def startItem = "|  ", endItem = "  |"
                 def repeatLength = startItem.length() + output.length() + endItem.length()
                 println("\n" + ("-" * repeatLength) + "\n" + startItem + output + endItem + "\n" + ("-" * repeatLength))
+                // @formatter:on
```

Comment:
One `@formatter:on` too much? Or why is this eclipse specific comment needed here?

Reply:
That is not needed. Probably an accidental paste. Removed again.